src/sentry/static/sentry/app/utils/withIssueTags.tsx

```diff
@@ -72,17 +89,20 @@ const withIssueTags = <P extends InjectedTagsProps>(
     },
 
     setAssigned() {
-      if (this.state.users && this.state.tags.assigned) {
-        const {tags, users} = this.state;
+      if (this.state.tags.assigned) {
+        const {tags, users, teams} = this.state;
         const usernames: string[] = users.map(getUsername);
-        usernames.unshift('me');
-
+        const teamnames: string[] = teams
+          .filter(getTeamName)
```

Comment:
i would inline getTeamName and return isMember instead.
`team => team.isMember`